src/main/java/edu/illinois/library/cantaloupe/async/ThreadPool.java

```diff
@@ -66,11 +64,11 @@ String getThreadNamePrefix() {
 
     private boolean isShutdown = false;
     private final ExecutorService lowPriorityPool =
-            Executors.newCachedThreadPool(new LowPriorityThreadFactory());
+            Executors.newThreadPerTaskExecutor(new LowPriorityThreadFactory());
```

Comment:
The API says newVirtualThreadPerTaskExecutor is equivalent to newThreadPerTaskExecutor with a factory producing virtual threads